spec/input/spec.yml

```
@@ -75,6 +75,11 @@ spec:
     $ref: "../integration/img/spec.yml"
 
 versions:
+  - before: 3.5.0
```

Should it be 3.5.1 here ?
Or, is it expected here to set that folder as required just for versions `<3.5.0`?  

```suggestion
  - before: 3.5.1
```

---

As in 3.5.0 the support for OTEL collector configuration was added into input packages looks good to me :+1: 

```
- version: 3.5.0
  changes:
    - ...
    - description: Add support for OTel collector configuration in input packages.
      type: enhancement
      link: https://github.com/elastic/package-spec/pull/975